packages/base/card-api.gts

```diff
@@ -565,7 +565,12 @@ class ContainsMany<FieldT extends FieldDefConstructor>
       await Promise.all(
         value.map(async (entry, index) => {
           if (primitive in this.card) {
-            return this.card[deserialize](entry, relativeTo, doc);
+            return this.card[deserialize](
+              entry,
+              relativeTo,
+              doc,
+              _identityContext,
```

Comment:
we can rename to `identityContext`. the leading `_` is to get lint to not complain about an unused var. now that we are using it, we should not use `_` prefix.

Reply:
Updated the name. It turned out it was in `Contains` and `ContainsMany` classes where we neglected to pass the identity context to deserialize.

Reply:
also, was this the only `Field.deserialize` that was missing the `identityContext` as a param? did you check the others to make sure they were all passing it as well?

Reply:
No there were at least 4 places I updated. Initially I wasn't going to update when it's a primitive field, but decided to remain consistent.

Reply:
> Updated the name. It turned out it was in `Contains` and `ContainsMany` classes where we neglected to pass the identity context to deserialize.

ah. probably we we not considering nested links within a contains when this was originally written which is why we had the disconnect